jobs/build/signed-compose/rpmdiff.py

```diff
@@ -33,18 +33,31 @@ def rpmdiffs_resolved(advisory):
     rpmdiffs = advisory.externalTests(test_type='rpmdiff')
     completed_diffs = []
     incomplete_diffs = []
+    failed_diffs = []
 
     print("Current RPM Diff status data:")
     pp(rpmdiffs)
 
     for diff in rpmdiffs:
-        if diff['attributes']['status'] in ['INFO', 'WAIVED', 'FAILED', 'NEEDS_INSPECTION']:
+        if diff['attributes']['status'] in ['INFO', 'WAIVED', 'NEEDS_INSPECTION']:
```

Comment:
`NEEDS_INSPECTION` is effectively `FAILED`. I don't know why there's a distinction.

Reply:
Not really. I played around the Errata API when implementing `elliott rpmdiff show`, finding out that  Errata does use more concrete statuses in its API (unlike what you see on the Web UI):
https://github.com/openshift/elliott/pull/86/files#diff-fb7e98bd08dd54fd472c838e06d6224cR75-R85